modules/nimbus-eth2/default.nix

```diff
@@ -135,11 +135,26 @@ in {
               ${trustedNodeUrl} \
               ${backfilling}'';
 
+            # trustedNodeSync =
```

Comment:
Let's not keep obsolete code in the repo (it won't be clear to future readers why this commented code exists).